_posts/2017-04-27-Rails-5-1-final.markdown

```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+title: 'Rails 5.1: Loving JavaScript, System Tests, Encrypted Secrets, and more'
+category: releases
+author: dhh
+published: true
+date: 2017-04-27 09:00:00 -09:00
+---
+In celebration of the 12th [RailsConf](http://railsconf.com) in Phoenix, Austin this week, we're proud to announce that Rails 5.1 is ready in its final form! We've spent [over 4,100 commits since Rails 5.0](https://github.com/rails/rails/compare/v5.0.0...5-1-stable/) making everything EASIER, SIMPLER, and, uhhh, FUNNER? (That's a RailsConf joke).
```

Comment:
http://guides.rubyonrails.org/5_1_release_notes.html The link is redirecting to edgeguides now, but will work after the release.